src/main/java/uk/gov/hmcts/et/common/model/ccd/CaseData.java

```diff
@@ -1107,7 +1109,7 @@ public class CaseData extends Et1CaseData {
     @JsonProperty("tseResponseHasSupportingMaterial")
     private String tseResponseHasSupportingMaterial;
     @JsonProperty("tseResponseSupportingMaterial")
-    private List<DocumentTypeItem> tseResponseSupportingMaterial;
+    private List<GenericTypeItem<DocumentType>> tseResponseSupportingMaterial;
```

Comment:
Could this be a breaking change in et-sya-api where we add a response?

Reply:
Good spot, it very well might be. We're trying to deprecate `DocumentTypeItem` in favour of `GenericTypeItem<DocumentType>` (soon to just be `TypeItem<DocumentType>`) pending those techdebt tickets for R1.2 (RET-3460).

I would suggest we move ahead with this though, callbacks has some heavy usage on this. I'll take a look at et-sya-api and update where necessary, should only be a case of renaming. (The types are basically the same, but since its inside a list, Java doesnt view them as interchangeable)

Reply:
Okay in that case lets do a major version bump for this